src/screens/MemberDetail/MemberDetailMocks.ts

```
@@ -1,317 +1,44 @@
+import { UPDATE_CURRENT_USER_MUTATION } from 'GraphQl/Mutations/mutations';
```

ESLint

[error] 'UPDATE_CURRENT_USER_MUTATION' is defined but never used.

(@typescript-eslint/no-unused-vars)